wremnants/theory_tools.py

```diff
@@ -19,11 +22,92 @@
 
 scale_tensor_axes = (axis_muRfact, axis_muFfact)
 
-pdfMap = {"nnpdf31" : {
+pdfMap = {
+        "nnpdf31" : {
             "name" : "pdfNNPDF31",
             "branch" : "LHEPdfWeight",
-            },
-        }
+            "combine" : "symHessian",
+            "entries" : 101,
+            "alphas" : ["LHEPdfWeight[101]", "LHEPdfWeight[102]"],
```

Comment:
Should there be an alphaRange given here?

Reply:
Yes but currently I'm not using it for anything (should be used in the combine scripts to scale the alphaS variations)